src/home/TabHome.tsx

```diff
@@ -123,7 +125,13 @@ function TabHome({ navigation }: Props) {
     // rest of feed to load unencumbered
     setTimeout(tryImportContacts, 500)
 
-    dispatch(trackPointsEvent({ activityId: 'create-wallet' }))
+    // stale reference to trackOnceActivities is intentional here to avoid
+    // queuing up multiple events with different id's in the case of track
+    // create-wallet failure + successful tracking of another activity in
+    // trackOnceActivities in the same app session
+    if (!trackOnceActivities['create-wallet']) {
+      dispatch(trackPointsEvent({ activityId: 'create-wallet' }))
+    }
   }, [])
```

Comment:
nit: We could even decouple this one entirely from the UI and manage it just from the saga (waiting for a created account event).

Reply:
yes i was also thinking about this, since you also brought it up i moved it to the saga level. i thought if we do this on the UI level then we can save some actions being dispatched...but doing it on the saga level is more neat and less error prone so it's probably better